jobbergate-cli/jobbergate_cli/end_to_end_testing/base.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class BaseEntity(ABC):
```

Comment:
This seems like a good opportunity to use the `Protocol` class introduced in Python 3.8. The Auth0 blog has a nice write-up describing it: https://auth0.com/blog/protocol-types-in-python/

Reply:
I'll wait a bit on that since the class can the refactored in the near future.